btk/catalog.py

```diff
@@ -41,23 +43,8 @@ def name(self):
     def get_raw_catalog(self):
         return self._raw_catalog
 
-    def apply_selection_function(self, selection_function, *kwargs):
```

Comment:
So in the end we are removing the selection function altogether ? (it is fine by me)

Reply:
Yeah I think so, I figured it would be easiest to keep to scope narrow if possible. If we later decide we really need the selection functions we can re-add them.